dts/bindings/gnss/seeed,air530z.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Jerónimo Agulló
+# SPDX-License-Identifier: Apache-2.0
+
+description: AIR530Z Seeed GNSS modem
+
+compatible: "seeed,air530z"
```

Comment:
it looks like the module itself is actually from LuatOS (`luatos`), not from Seeed, so I think it's worth updating (although I realize it's a bit tedious at this point, so other on this thread should feel free to disagree :))? BTW it would also likely make sense to provide a shield definition for the "Grove GPS". 

Also curious if the driver would also be compatible with Air530 flavour (which Grove GPS used to use, IIUC)?

Reply:
ohh you are rigth! I made a mistake since the module (not the sensor) was shipped by Seeed and most of the information is available only in chineese. I will refract the code, it is straightforward

Reply:
I have just done a refractor of the whole code changing "seeed" to "luatos". I have built and flashed the application to be sure that nothing was broken and everything was running smoothly. Thus, it is ready :)

Reply:
@kartben regarding the difference between Air530 and Air530z. They are totally different modules, while Air530 uses `PGKC` message, the Air530z using `PCAS` messages.  As I read on Internet, the Air530z is a newest module.

It was a really headache because I though that both were equal. Actually, I supposed to have ordered a Air530 module, but I received the Air530z.